### PR TITLE
Feat: Added Typography-Conclusion

### PR DESCRIPTION
**Pull request by Abhijeet43**
Added Typography and conclusion pages

Demo Link: https://deploy-preview-3--nova-ui.netlify.app/getstarted/typography/typography

### CODE REVIEW

**Review by Ayushi20-19 — APPROVED** [review]
good to go

File: getStarted/typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  font-size: var(--font-size-3);
+}
+
+.small-text {
+  font-size: var(--font-size-4);
+}
+
+.x-small-text {
+  font-size: var(--font-size-5);
+}
```
Comment (Husain01) [inline getStarted/typography/typography.css:20]: The sizes are not getting applied. Try importing the stylesheet where these variables are defined.

File: getStarted/typography/typography.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  font-weight: var(--font-weight-2);
+}
+
+.font-weight-500 {
+  font-weight: var(--font-weight-3);
+}
+
+.font-weight-300 {
+  font-weight: var(--font-weight-3);
+}
```
Comment (Husain01) [inline getStarted/typography/typography.css:38]: Same with the font-weights. Not getting applied.

**Review by Husain01 — CHANGES_REQUESTED** [review]

File: getStarted/typography/typography.css (hunk 1 above)
Comment (Abhijeet43) [reply getStarted/typography/typography.css:20]: Thanks for pointing fixing it now

**Review by mohammed-sas — APPROVED** [review]
pr good to merge

**Pull request merged**